docs/AboutUs.md

```diff
@@ -31,14 +31,16 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 I am an eager Y2 computer science undergraduate with the desire to learn and improve myself in this field. I aim to delve deeper into the world of software engineering and cybersecurity which address and solve real world problems with the focus on privacy and data security.
 
-### Johnny Doe
+### Edward Alvin
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/Edward_Alvin.jpeg" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
```

Comment:
The portfolio link I believe should be kept as it is supposed to link to the PPP